storage/aptosdb/src/ledger_db/transaction_accumulator_db.rs

```diff
@@ -53,27 +53,6 @@ impl TransactionAccumulatorDb {
     pub(crate) fn write_schemas(&self, batch: SchemaBatch) -> Result<()> {
         self.db.write_schemas(batch)
     }
-
-    pub(crate) fn revert_transaction_accumulator(
```

Comment:
The main thing I'm wondering about all the reversion bits I merged in, is why are there so many deletions.

Reply:
These methods have been added by us (namely @0xmovses), but the changes in #28 make them surplus to requirements, so I decided to remove them and reduce the delta with upstream. Anyway, this should be subject for a less hectic review over a smaller diff in #28, which will need to be properly updated after this sync.

Reply:
Or we may leave #28 in, but then someone else needs to go over them because we're effectively rolling those changes (is it all of them, BTW?) into #30.